dddb/src/lib.rs

```diff
@@ -0,0 +1,11 @@
+use pyo3::prelude::*;
+use isg_4real::*;
+
+#[pymodule]
+fn dddb(_py: Python<'_>, m: &PyModule) -> PyResult<()> {
+    let submodule = PyModule::new(_py, "video")?;
+    submodule.add_function(wrap_pyfunction!(encode, m)?)?;
```

Comment:
obviously i've never worked with pyo3 before but why's the interface

```py
dddb.video.decode({'in_path': 'test.mp4', 'out_path': 'test-new.txt'})
```

and not
```py
dddb.video.decode(in_path = 'test.mp4', out_path = 'test.txt')
```

not a big concern but why does it do this lol

Reply:
allows us to pass in an arbitrary argument set